components/Snackbar/src/MDCSnackbarManager.h

```diff
@@ -385,6 +393,12 @@
  */
 + (void)setButtonTitleColor:(nullable UIColor *)titleColor forState:(UIControlState)state;
 
+/**
+ Bound to @c adjustsFontForContentSizeCategory on the @c defaultManager instance.
+ */
+@property(class, nonatomic, readwrite, setter=setAdjustsFontForContentSizeCategory:)
```

Comment:
I'm on the fence about this, but I'd really rather leave it out. Since class/static APIs are on their way out and we prefer clients to use SnackbarManager.defaultInstance instead, I don't think it's worth adding a new API.